File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entity/GregorianCalendarEvent.java
package com.byagowi.persiancalendar.entity;

import calendar.CivilDate;

/**
 * PersianCalendarEvent
 *
 * @author ebraminio
 */
public class GregorianCalendarEvent extends AbstractEvent<CivilDate> {
    public GregorianCalendarEvent(CivilDate date, String title, boolean holiday) {
        this.date = date;
        this.title = title;
        this.holiday = holiday;
    }
}

File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entity/GregorianCalendarEvent.kt
package com.byagowi.persiancalendar.entity

import calendar.CivilDate

/**
 * PersianCalendarEvent
 *
 * @author ebraminio
 */
class GregorianCalendarEvent(val date: CivilDate, title: String,
                             holiday: Boolean) : AbstractEvent() {

    init {
        this.title = title
        this.holiday = holiday
    }
}